server/routes/notifications.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  readNotification,
+  getAllNotifications,
+  getUnreadNotifications,
+} = require("../controllers/notification");
+
+router.route("/create").post(createNotification);
```

Comment:
```suggestion
router.route("/create").post(protect, createNotification);
```
Ditto throughout!

Note: The `protect` middleware only allows authenticated users to access a route.